tests/baselines/reference/tsbuild/moduleSpecifiers/synthesized-module-specifiers-across-projects-resolve-correctly.js

```diff
@@ -134,296 +134,33 @@ Output::
 
 [[90m12:00:26 AM[0m] Building project '/src/src-types/tsconfig.json'...
 
-[[90m12:00:33 AM[0m] Project 'src/src-dogs/tsconfig.json' is out of date because output file 'src/src-dogs/dog.js' does not exist

```

Comment:
Can you fix the tests to modify lib file so there are no errors are we are really testing the scenario.

Reply:
Created #48974